### PR TITLE
Update localization chapter

### PR DESCRIPTION
**Pull request by jason-p-pickering**

### CODE REVIEW

File: src/commonmark/en/content/implementation/localization-of-dhis2.md [hunk 1]
```diff
@@ -27,96 +31,96 @@ In French the same key/value pairs would appear as follows
...
+## DHIS 2 i18n tool
 
 <!--DHIS2-SECTION-ID:dhis2-i18n-translations-tool-->
 
-The i18n resource editor is Java desktop application which can be
-downloaded from http://www.dhis2.org/downloads . It requires that you
-have checked out the DHIS2 source code from Github (check out
-http://www.dhis2.org/development if necessary) and have a Java Runtime
-Environment installed on your computer. On Windows, simly unpack the ZIP
+The i18n resource editor is Java desktop application which can beYour translations
```
Comment (Philip-Larsen-Donnelly) [inline src/commonmark/en/content/implementation/localization-of-dhis2.md]: There appear to be several chunks of leftover text in this paragraph

File: src/commonmark/en/content/implementation/localization-of-dhis2.md [hunk 2]
```diff
@@ -27,96 +31,96 @@ In French the same key/value pairs would appear as follows
...
-    DHIS2 source code repository, followed by OK.
+    DHIS 2 source code repository, followed by OK.
     
     
     ![](resources/images/i18n/i18n_tool_startup.png)
-
-2.  Next, select the destination locale which you will translate strings
-    into. Remember that if you want to create or add to a general
-    language translation, select e.g. "Portuguese - \[pt\]. If you want
+Your translations
```
Comment (Philip-Larsen-Donnelly) [inline src/commonmark/en/content/implementation/localization-of-dhis2.md]: This sentence repeats four times! I think it shouldn't be here at all - instead you should have a blank line, then the next numbered item.

**Review by Philip-Larsen-Donnelly — CHANGES_REQUESTED** [review]
Some errors with extra text in the document. See comments.

File: src/commonmark/en/content/implementation/localization-of-dhis2.md (hunk 2 above)
Comment (jason-p-pickering) [reply src/commonmark/en/content/implementation/localization-of-dhis2.md]: Got it. Will look at it.

**Review by Philip-Larsen-Donnelly — APPROVED** [review]

**Pull request merged**